.claude/audit/scripts/diff.py

```
@@ -0,0 +1,110 @@
+def prop_token(name):
+    return name.split("_", 1)[1] if name.startswith(("get_", "set_")) else name
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--census", default=DEFAULT_CENSUS)
+    ap.add_argument("--skill-dir", default=DEFAULT_SKILL_DIR)
+    ap.add_argument("--out", default=DEFAULT_OUT)
+    args = ap.parse_args()
+
+    data = json.load(open(args.census, encoding="utf-8"))
```

Ruff (0.15.14)

[warning] Use a context manager for opening files

(SIM115)